dist/components/app.jsx

```diff
@@ -4,23 +4,70 @@
 /* eslint-disable react/destructuring-assignment */
 import React from 'react';
 import axios from 'axios';
-<<<<<<< HEAD
 import styled from 'styled-components';
 // import Listings from './Listings.jsx';
+import Calendar from './calendar/index.jsx';
 
 const Wrapper = styled.section`
   border: solid;
   width: 25%;
 `;
 
 const Price = styled.section`
-  border: solid;
   width: 25%;
 `;
-=======
-import Listings from './Listings.jsx';
-import Calendar from './calendar/index.jsx';
->>>>>>> cf0b8c162dabf6747cf1e12743b1d3fb36dd3836
+
```

Comment:
Fingers crossed that it isn't too difficult to import all of the styling from all 5 services